sonar-core/src/main/java/org/sonar/core/config/WebhookProperties.java

```diff
@@ -51,6 +51,9 @@
   private static final String DESCRIPTION = "Webhooks are used to notify external services when a project analysis is done. " +
     "An HTTP POST request including a JSON payload is sent to each of the first ten provided URLs. <br/>" +
     "Learn more in the <a href=\"https://redirect.sonarsource.com/doc/webhooks.html\">Webhooks documentation</a>.";
+  private static final String URL_DESCRIPTION = "Server endpoint that will receive the webhook payload, for example 'http://my_server/foo'. " +
+    "In the case that HTTP basic credentials are defined, using HTTPS is recommended to avoid man in the middle attacks. " +
```

Comment:
`If HTTP Basic authentication is used, HTTPS is recommended to avoid man in the middle attacks.`